blog/_posts/2017-03-03-ruby-conf-australia.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Ruby Conf Australia
+layout: post
+created_at: Fri Mar 03 2017
+permalink: blog/2017-03-03-ruby-conf-australia
+current: blog
+author: Ramya
+twitter: ramya690
+categories: student-posts
+---
+
+I had a privilege to attend RubyConf held in Melbourne last month as a part of the Rails Girls Summer of Code program. The Venue was set at the Melbourne Convention Hall and it was a great opportunity to meet so many interesting people in the Ruby community. Also a good learning experience. 
+
+
+![Ruby Conf Australia](/img/blog/2017/2017-03-03-rubyconf-logo
+-au.jpg)<font color="grey"><small><i>Ruby Conf Australia</i></small></font> 
```

Comment:
could you add an image credit here? (i.e. who took the picture, as explained in the [contributing guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images))